### ISSUE TITLE
Update via Homebrew fails

### ISSUE THREAD

**Issue by Dschee** [yonaskolb/Beak#24]
I have beak 0.3.3 installed on my machine (macOS 10.13.4 & Xcode 9.3) but get an error when trying to upgrade to the latest version (0.3.5). Here's the output:

```
$ brew upgrade beak
Updating Homebrew...
==> Auto-updated Homebrew!
Updated Homebrew from 7166289a to 714bf515.
Updated 2 taps (homebrew/core, caskroom/cask).
==> Updated Formulae
annie                       geth                        libphonenumber              pgpool-ii                   sonarqube                   zsh

==> Upgrading 1 outdated package, with result:
yonaskolb/beak/beak 0.3.3 -> 0.3.5
==> Upgrading yonaskolb/beak/beak
==> Downloading https://github.com/yonaskolb/Beak/archive/0.3.5.tar.gz
==> Downloading from https://codeload.github.com/yonaskolb/Beak/tar.gz/0.3.5
######################################################################## 100.0%
==> Building Beak
==> swift build --disable-sandbox -c release -Xswiftc -static-stdlib
Last 15 lines from /Users/Me/Library/Logs/Homebrew/beak/01.swift:
Resolving https://github.com/kylef/Spectre.git at 0.8.0
Cloning https://github.com/drmohundro/SWXMLHash.git
Resolving https://github.com/drmohundro/SWXMLHash.git at 4.6.0
Cloning https://github.com/antitypical/Result.git
Resolving https://github.com/antitypical/Result.git at 3.2.4
Cloning https://github.com/norio-nomura/SourceKit.git
Resolving https://github.com/norio-nomura/SourceKit.git at 1.0.1
Cloning https://github.com/Carthage/Commandant.git
Resolving https://github.com/Carthage/Commandant.git at 0.13.0
Cloning https://github.com/kylef/PathKit.git
Resolving https://github.com/kylef/PathKit.git at 0.9.1
Cloning https://github.com/apple/swift-package-manager
Resolving https://github.com/apple/swift-package-manager at 0.2.0
Cloning https://github.com/norio-nomura/Clang_C.git
Resolving https://github.com/norio-nomura/Clang_C.git at 1.0.2

If reporting this issue please do so to (not Homebrew/brew or Homebrew/core):
yonaskolb/beak

Error: curl failed!
curl: (22) The requested URL returned error: 422 Unprocessable Entity
curl: (3) <url> malformed
```

**Comment by yonaskolb** [comment]
This is an issue in homebrew due to changes in Xcode 9.3.
Possible homebrew fix here https://github.com/Homebrew/brew/pull/4147

**Comment by yonaskolb** [comment]
Fixed in 0.4.0
let me know if you have any more issues @Dschee
